server/src/business-layer/utils/BookingUtils.ts

```diff
@@ -0,0 +1,76 @@
+import { Timestamp } from "firebase-admin/firestore"
+import { LodgePricingTypeValues } from "./StripeProductMetadata"
+
+// Need to validate the booking date through a startDate and endDate range.
+/**
+ * @deprecated do not use, exported for testing purposes
+ */
+export const _earliestDate = new Date()
```

Comment:
We'll have to restart the relevant servers so this updates when a new year rolls over, but that's probably OK :)

Reply:
actually true that - can see thing being a problem